packages/core/useMouse/index.ts

```diff
@@ -97,6 +99,11 @@ export function useMouse(options: UseMouseOptions = {}) {
       [x.value, y.value] = result
       sourceType.value = 'mouse'
     }
+
+    if (window) {
+      _prevScrollX = window.scrollX
+      _prevScrollY = window.screenY
```

Comment:
[#4348](https://github.com/vueuse/vueuse/pull/4348)
It was my mistake. I'm sorry.